src/main/resources/api.yaml

```diff
@@ -207,3 +312,35 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+  ProjectMemberDto:
+    type: object
+    properties:
+      userId:
+        type: integer
+        format: int64
+      userDetails:
+        $ref: '#/definitions/UserDetailsDto'
+      role:
+        $ref: '#/definitions/UserRoleDto'
+  UserRoleDto:
+    type: string
+    enum:
+      - admin
+      - member
+  UserDetailsDto:
+    type: object
+    properties:
+      firstName:
+        type: string
+      lastName:
+        type: string
+      email:
+        type: string
+  UserIdentificationDto:
+    type: object
+    properties:
+      email:
+        type: string
+      id:
+        type: integer
+        format: int64
```

Comment:
chodzi tutaj o to, że są dostępne dwie metody na dodanie użytkownika, czy o to, że oba pola muszą być wypełnione?

Reply:
Dwie metody na zidentyfikowanie użytkownika, albo przez ID albo przez email. Jeśli oba są podane to muszą się zgadzać :) dodam description, w którym jest to opisane